x-pack/plugins/watcher/public/sections/watch_edit/components/watch_edit_detail/watch_edit_detail.html

```diff
@@ -50,17 +50,13 @@
 </form>
 
 <div class="kuiFormSection">
-  <label class="kuiFormLabel">
-    Watch JSON
-    (<a
-      href="{{ watchEditDetail.documentationLinks.watcher.putWatchApi }}"
-      target="_blank"
-      rel="noopener noreferrer"
-      data-test-subj="lnkPutWatchApiDoc"
-    >
-    Syntax
-    </a>)
-  </label>
+  <label
+    class="kuiFormLabel"
+    i18n-id="xpack.watcher.sections.watchEdit.detail.watchJsonLinkText"
+    i18n-default-message="Watch JSON({linkOpenTag}Syntax{linkCloseTag})"
+    i18n-values="{ linkOpenTag: '<a href=\'{{ watchEditDetail.documentationLinks.watcher.putWatchApi }}\' target=\'_blank\' rel=\'noopener noreferrer\' data-test-subj=\'lnkPutWatchApiDoc\'>',
+        linkCloseTag: '</a>' }"
```

Comment:
please do not use linkOpenTag and linkCloseTag anymore. We deprecated such style.